package.json

```diff
@@ -6,12 +6,7 @@
   "author": "Google",
   "license": "Apache-2.0",
   "type": "module",
-  "keywords": [
-    "amp",
-    "stories",
-    "storytelling",
-    "wordpress"
-  ],
+  "keywords": ["amp", "stories", "storytelling", "wordpress"],
```

Comment:
Can you undo changes to this file?

Curious: what's your `npm` version?

Trying to figure out why such changes happen from time to time.